IdentityCore/src/cache/MSIDCacheAccessor.h

```diff
@@ -96,8 +82,18 @@
                                              error:(NSError **)error;
 
 - (BOOL)clearCacheForAccount:(MSIDAccountIdentifier *)account
+                   authority:(MSIDAuthority *)authority
                     clientId:(NSString *)clientId
+                    familyId:(NSString *)familyId
                      context:(id<MSIDRequestContext>)context
                        error:(NSError **)error;
 
+- (BOOL)validateAndRemoveRefreshToken:(MSIDBaseToken<MSIDRefreshableToken> *)token
```

Comment:
nit: is it really "validating" anything?

Reply:
It validates that the refresh token hasn't been changed by other app :)